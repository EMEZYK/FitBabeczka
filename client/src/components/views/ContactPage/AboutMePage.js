import React from "react";
import {
  AboutMeWrapper,
  PhotoAbout,
  TextContainer,
  Greeting,
  DescriptionAbout,
  Page,
} from "./AboutMePage.styled";

const AboutMePage = () => {
  return (
    <Page>
      <AboutMeWrapper
        direction="row"
        padding="0px 40px"
        border="12px solid #956682"
        flex-wrap="wrap"
      >
        <PhotoAbout src={"/photos/adminPhoto.png"} alt="photo-about" />
        <TextContainer>
          <Greeting data-testid="greeting">Hej! Jestem Ewelina</Greeting>
          <DescriptionAbout>
            Serdecznie witam Cię na mojej kulinarnej stronie i zapraszam do
            wspólnego pieczenia! <br /> <br />
            Jeśli tak jak ja, uwielbiasz przygotowywać różnego rodzaju
            słodkości, a jednocześnie chcesz zadbać o swoje zdrowie, to ten blog
            skierowany jest właśnie do Ciebie. Znajdziesz tu mnóstwo przepisów
            na wypieki (ciasta, babeczki, desery i wiele innych), w których
            najbardziej toksyczne dla organizmu składniki, staram się zastąpić
            zdrowymi zamienikami.
            <br />
            <br />
            Kilka słów o mnie: Pasję do tworzenia słodkich wypieków odnalazłam w
            sobie już w dzieciństwie. Z czasem, coraz większą uwagę zaczęłam
            zwracać na aspekt zdrowotny moich słodkości. Stąd pojawił się pomysł
            na FitBabeczkę. W międzyczasie w moim życiu zrodziła się nowa pasja
            - programowanie. Cieszę się, że właśnie w tym miejscu, mogę połączyć
            oba moje zainteresowania i podzielić się z Tobą efektami mojej
            pracy.
            <br />
            <br />
            Masz pytania? Napisz do mnie na adres: ewelinamezykmail@gmail.com.
            <br />
            <br />
            Piecz, delektuj się słodkościami i żyj zdrowo!
          </DescriptionAbout>
        </TextContainer>
      </AboutMeWrapper>
    </Page>
  );
};

export default AboutMePage;
