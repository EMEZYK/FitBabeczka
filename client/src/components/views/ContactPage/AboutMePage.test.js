import { render, cleanup } from "@testing-library/react";
import { screen } from "@testing-library/dom";
import "@testing-library/jest-dom";
import AboutMePage from "../ContactPage/AboutMePage";

describe("AboutMePage components", () => {
  afterEach(cleanup);

  test("Should take a snapshot',", () => {
    const { asFragment } = render(<AboutMePage />);
    expect(asFragment(<AboutMePage />)).toMatchSnapshot();
  });

  test("Image is rendered", () => {
    render(<AboutMePage />);
    const image = screen.getByAltText("photo-about");
    expect(image).toHaveAttribute("src", "/photos/adminPhoto.png");
  });

  test("Renders greeting =", () => {
    render(<AboutMePage />);
    const greeting = screen.queryByTestId("greeting");
    expect(greeting).toHaveTextContent("Hej! Jestem Ewelina");
  });
});
