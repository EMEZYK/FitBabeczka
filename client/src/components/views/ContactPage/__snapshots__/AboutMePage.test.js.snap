// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMePage components Should take a snapshot', 1`] = `
<DocumentFragment>
  <div
    class="sc-ftvSup dIGkDo"
  >
    <div
      class="sc-gsnTZi sc-papXJ fRA-dqe kcfMkN"
      direction="row"
      width="84%"
    >
      <img
        alt="photo-about"
        class="sc-eCYdqJ sc-jqUVSM fjibNq cOOeGx"
        src="/photos/adminPhoto.png"
      />
      <div
        class="sc-kDDrLX frtriJ"
      >
        <p
          class="sc-bczRLJ sc-iqcoie hsohjl dANOlj"
          data-testid="greeting"
        >
          Hej! Jestem Ewelina
        </p>
        <p
          class="sc-bczRLJ sc-crXcEl hsohjl dWAOqs"
        >
          Serdecznie witam Cię na mojej kulinarnej stronie i zapraszam do wspólnego pieczenia! 
          <br />
           
          <br />
          Jeśli tak jak ja, uwielbiasz przygotowywać różnego rodzaju słodkości, a jednocześnie chcesz zadbać o swoje zdrowie, to ten blog skierowany jest właśnie do Ciebie. Znajdziesz tu mnóstwo przepisów na wypieki (ciasta, babeczki, desery i wiele innych), w których najbardziej toksyczne dla organizmu składniki, staram się zastąpić zdrowymi zamienikami.
          <br />
          <br />
          Kilka słów o mnie: Pasję do tworzenia słodkich wypieków odnalazłam w sobie już w dzieciństwie. Z czasem, coraz większą uwagę zaczęłam zwracać na aspekt zdrowotny moich słodkości. Stąd pojawił się pomysł na FitBabeczkę. W międzyczasie w moim życiu zrodziła się nowa pasja - programowanie. Cieszę się, że właśnie w tym miejscu, mogę połączyć oba moje zainteresowania i podzielić się z Tobą efektami mojej pracy.
          <br />
          <br />
          Masz pytania? Napisz do mnie na adres: ewelinamezykmail@gmail.com.
          <br />
          <br />
          Piecz, delektuj się słodkościami i żyj zdrowo!
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
